### ISSUE TITLE
Use of embedded external dependencies

### ISSUE THREAD

**Issue by olivier-roussel** [SofaDefrost/SoftRobots.Inverse#10]
Following SOFA issue: https://github.com/sofa-framework/sofa/issues/4221
Here is the list of external dependencies embedded in the plugin.

| Lib name | Type | Location in code | License  |
| --- | --- | --- | --- |
| qpOASES | shared | `extlibs` | LGPL 2.1 |

Should we consider moving to an external dependency instead for qpOASES or not ?

**Comment by alxbilger** [comment]
Yes! And this would also be an opportunity to try other libraries. If I understood correctly, this one is not the most efficient

**Comment by olivier-roussel** [comment]
Great.
Agreed to try other libraries. ProxQP seems to be a good candidate, it has been benchmarked against others recently (see https://github.com/Simple-Robotics/proxqp_benchmark for results).

**Comment by alxbilger** [comment]
Yes, this is the one we had in mind.

**Comment by olivier-roussel** [comment]
Closed as solved by #22
